Polygon_mesh_processing/test/Polygon_mesh_processing/CMakeLists.txt

```diff
@@ -76,6 +76,7 @@ if (EIGEN3_FOUND)
   create_single_source_cgal_program("remeshing_test.cpp" )
   create_single_source_cgal_program("measures_test.cpp")
   create_single_source_cgal_program("triangulate_faces_test.cpp")
+  create_single_source_cgal_program("test_pmp_remove_border_edge.cpp")
```

Comment:
There is a CMake warning, about a missing .cpp file:
```
-- Configuring test in test/Polygon_mesh_processing
-- Requested component: MPFR
-- Requested component: GMPXX
-- Requested component: GMP
-- Boost version: 1.60.0
CMake Warning (dev) at Installation/cmake/modules/CGAL_CreateSingleSourceCGALProgram.cmake:72 (message):
  The executable test_pmp_remove_border_edge will not be created because the
  source file
  /home/lrineau/Git/cgal/Polygon_mesh_processing/test/Polygon_mesh_processing/test_pmp_remove_border_edge.cpp
  does not exist.
Call Stack (most recent call first):
  Polygon_mesh_processing/test/Polygon_mesh_processing/CMakeLists.txt:79 (create_single_source_cgal_program)
This warning is for project developers.  Use -Wno-dev to suppress it.

```